tools/bowtie2/bowtie2_wrapper.xml

```diff
@@ -437,17 +437,40 @@
         <data format="fastqsanger" name="output_unaligned_reads_l" label="${tool.name} on ${on_string}: unaligned reads (L)" >
             <filter>library['unaligned_file'] is True</filter>
             <actions>
-                <action type="format">
-                    <option type="from_param" name="library.input_1" param_attribute="ext" />
-                </action>
+                <conditional name="library.type">
+                    <when value="single">
```

Comment:
That is the reason I called it `values` and not `value`. I you need a comma you can fallback to the old style ;)